src/include/miopen/any_solver.hpp

```diff
@@ -56,10 +56,10 @@ struct AnySolver
         assert(ptr_value != nullptr);
         return ptr_value->IsTunable();
     };
-    bool TestSysDbRecord(const ConvolutionContext& ctx, const DbRecord& record) const
+    bool TestPerfCfgParams(const ConvolutionContext& ctx, std::string params) const
```

Comment:
```suggestion
    bool TestPerfCfgDeserialize(const ConvolutionContext& ctx, const std::string& params) const
```

Reply:
This is testing if the params are a valid performance config, so this function name is partially misleading.
True, Deserialize is called to get the config, IsValidPerformanceConfig is afterwards called on the config to validate the params.

Reply:
Update formal parameters to require a const string pointer.